recipes/fontconfig/all/conanfile.py

```diff
@@ -65,21 +75,25 @@ def build(self):
 
     def package(self):
         self.copy("COPYING", dst="licenses", src=self._source_subfolder)
-        autotools = self._configure_autotools()
-        autotools.install()
+        with tools.run_environment(self):
+            autotools = self._configure_autotools()
+            autotools.install()
         os.unlink(os.path.join(self.package_folder, "lib", "libfontconfig.la"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
         for f in glob.glob(os.path.join(self.package_folder, "bin", "etc", "fonts", "conf.d", "*.conf")):
             if os.path.islink(f):
                 os.unlink(f)
 
-
     def package_info(self):
         self.cpp_info.libs = ["fontconfig"]
         if self.settings.os == "Linux":
             self.cpp_info.system_libs.extend(["m", "pthread"])
         self.cpp_info.names["cmake_find_package"] = "Fontconfig"
         self.cpp_info.names["cmake_find_package_multi"] = "Fontconfig"
 
-        self.env_info.FONTCONFIG_FILE = os.path.join(self.package_folder, "bin", "etc", "fonts", "fonts.conf")
-        self.env_info.FONTCONFIG_PATH = os.path.join(self.package_folder, "bin", "etc", "fonts")
+        fontconfig_file = os.path.join(self.package_folder, "bin", "etc", "fonts", "fonts.conf")
```

Comment:
Is this path ok for Windows: `/` vs `\`?

Reply:
Windows is not supported yet in this recipe (but I'll push a commit adding support for MinGW). All others recipe add path in env variables without replacing `\\`, why would it not work for fontconfig?

Reply:
Some environment variables are used by winbash environments which might misinterpret the `\`.
That's why I often add a `.replace("\\", "/")`.

Reply:
So all recipes creating or appending env var should be updated to replace `\\` by `/`?

Reply:
Depends on whether they are used by a bash script or not.
I wouldn't make it a general rule.

Reply:
I am not entirely sure it's a fully valid fix. for certain FS having a backslash in the filename is okay.

Reply:
This is just protection against that *one dirty bash script* that uses environment variables.

Reply:
so I'm still not sure if I should replace baskslashes

Reply:
It's ok I think.
My remark was hypothetical, it will fail when used under winbash.
If it ever causes a problem, it can be fixed then.